CONTRIBUTING.md

```diff
@@ -66,6 +73,10 @@ We write
 - Operators are surrounded by space, for example `n*m` should be written `n * m`.
 This particular example can be problematic if matrix.v is imported because then, `M *m N` is matrix product.
 
+### Statement-macros
+- There is quite a number of "macros" to state lemmas, like `commutative`, `associative`,... (see `ssrfun.v`)
```

Comment:
Does it mean these macros should be used consistently across Mathcomp? Because right now it's not precisely the case. E.g. `cat0s`, `cats0`, `catA` in `seq` or a bunch of lemmas for set operations in `finset`. I can open an issue about that if you feel this is something worth doing.

Reply:
I rephrase to avoid giving the impression that there is a rule here.